### PR TITLE
More changes to get to production stage.

### PR DESCRIPTION
**Pull request by arkilic**

### CODE REVIEW

File: metadataclient/commands.py [hunk 1]
```diff
@@ -715,8 +708,7 @@ def bulk_insert_events(event_descriptor, events, validate=False):
     chunk_size = 500
     data_len = len(events)
     chunk_count = data_len // chunk_size + bool(data_len % chunk_size)
-    chunks = (list(events)[(j*chunk_size):((j+1)*chunk_size)] for j in range(int(chunk_count)))
-    
+    chunks = utils.grouper(events, chunk_count)
     for c in chunks:
         payload = ujson.dumps(list(c))
```
Comment (tacaswell) [inline metadataclient/commands.py]: You might need to make this line

```python
ujson.dumps([_c for _c in c if _c is not None])
```

**Comment by hhslepicka** [comment]
So far so good! The web gui will be left for the future.

**Comment by arkilic** [comment]
That's a sudden loss of enthusiasm lol. Too busy fixing the beamline... Sheesh

**Comment by arkilic** [comment]
@hhslepicka fixes the problems we had last week and passes all tests. Take a look and make sure we have a full functioning master (finally.)

**Pull request merged**